packages/apps/gamepadcalibration/package.mk

```
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2024 ROCKNIX (https://github.com/ROCKNIX)
+
+PKG_NAME="gamepadcalibration"
+PKG_VERSION="973bf8c4d72068e3e2383450c76b0cb2fa078cb1"
+PKG_LICENSE="MIT"
+PKG_SITE="https://github.com/cdeletre/GPcal"
+PKG_URL="${PKG_SITE}.git"
```

```suggestion
PKG_URL="https://github.com/cdeletre/GPcal/archive/${PKG_VERSION}.tar.gz"
```

Can you change to using github archive as we are trying to move away from git repositories.

---

Sure, will do.